package/contents/ui/lib/ConfigSpinBox.qml

```diff
@@ -30,7 +28,36 @@ RowLayout {
 		value: plasmoid.configuration[configKey]
 		// onValueChanged: plasmoid.configuration[configKey] = value
 		onValueChanged: serializeTimer.start()
-		maximumValue: 2147483647
+		to: 2147483647
+		
+		property string suffix: ""
+		
+		/* copied from https://doc.qt.io/qt-6/qml-qtquick-controls-spinbox.html
+		 * because QtQuick 6.0 does not include properties that where in QtQuick.Controls 1.2 */
+		property int decimals: 2
+		property real realValue: value / decimalFactor
+		readonly property int decimalFactor: Math.pow(10, decimals)
```

Comment:
God do I ever hate QQC2 for forcing us to implement this logic every time just for a floating point SpinBox.

I've got a complete QQC2 LibConfig which I plan to use.

I found that trying to parse out the suffix when editing was a pain in the ass, so I just created some overlaid labels before/after the input text.

* https://github.com/Zren/plasma-applet-lib/tree/master/package/contents/ui/libconfig
* https://github.com/Zren/plasma-applet-lib/blob/master/package/contents/ui/libconfig/SpinBox.qml